react/typings/vtex.render-runtime.d.ts

```diff
@@ -2,6 +2,7 @@
 declare module 'vtex.render-runtime' {
   export const useRuntime: () => {
     account: string
+    page: string
```

Comment:
We can remove this file and run `vtex setup --typings`

This will install the app types that are working nice 👌

Reply:
It worked perfectly, Thanks for the advice.